games-board/netmaumau/netmaumau-0.15.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2015 Julian Ospald <hasufell@posteo.de>, Heiko Schaefer <heiko@rangun.de>
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit qmake-utils eutils vcs-snapshot
+
+MY_P=nmm-qt-client${PV}
+
+DESCRIPTION="Client for games-server/netmaumau, the popular card game Mau Mau"
+HOMEPAGE="http://sourceforge.net/projects/netmaumau"
+SRC_URI="https://github.com/velnias75/NetMauMau-Qt-Client/archive/V${PV}.tar.gz -> ${P}-client.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="no-espeak"
```

Comment:
> As you know I hate to make new or edit PR's

Why's that? It's the most common operation after ```git commit```.
You can do it either via an interactive rebase:

```
git rebase -i HEAD~2
# change 'pick' to 'e' at the offending commit
# save editor and exit
# git rewinds to that commit now
# fix the games-board/netmaumau ebuild
git add games-board/netmaumau
git commit --amend
git rebase --continue
```

 or just do a mixed reset and repeat both commits:

```
git reset --mixed HEAD~2
# fix games-board/netmaumau ebuild
git add games-board/netmaumau
git commit -ma '<commit message>'
git add games-server/netmaumau
git commit -ma '<commit message>'
```